settings.gradle

```diff
@@ -0,0 +1,11 @@
+
+plugins {
```

Comment:
The com.gradle.build-scan plugin is now called 'com.gradle.enterprise' and must be added in the settings file and not the build file

Reply:
What are we using this for?